src/Management/test/Endpoint.Test/Actuators/Health/EndpointMiddlewareTest.cs

```diff
@@ -18,9 +18,9 @@ public sealed class EndpointMiddlewareTest : BaseTest
 {
     private readonly Dictionary<string, string?> _appSettings = new()
     {
-        ["management:endpoints:enabled"] = "true",
-        ["management:endpoints:path"] = "/cloudfoundryapplication",
-        ["management:endpoints:health:enabled"] = "true"
+        { "Management:Endpoints:Enabled", "true" },
+        { "Management:Endpoints:Path", "/cloudfoundryapplication" },
+        { "Management:Endpoints:Health:Enabled", "true" }
```

Comment:
Why switch back to the old-style dictionary syntax here?

Reply:
I intended to make the file consistent, but I guess I did it backwards